zerver/tornadoviews.py

```diff
@@ -54,6 +62,7 @@ def get_events_backend(request, user_profile, handler,
                        dont_block = REQ(default=False, validator=check_bool),
                        narrow = REQ(default=[], validator=check_list(None)),
                        lifespan_secs = REQ(default=0, converter=int)):
+    # type: (HttpRequest, UserProfile, AsyncDjangoHandler, Optional[Client], Optional[int], Optional[str], bool, bool, Optional[str], bool, List[Optional[Tuple[str, str]]], int) -> Union[HttpResponse, _RespondAsynchronously]
```

Comment:
I'm pretty sure `event_types` should be an Optional[List[str]], not an Optional[str].

Also I think `narrow` should just be a `List[Tuple[str, str]]`; the None argument passed to `check_list` just means it's not validating the type of the elements in the list at all.